examples/rust/battleship/web/client/src/game.rs

```diff
@@ -103,7 +103,7 @@ fn check_taken(x: usize, y: usize, span:usize, dir:usize, taken:Taken) -> bool {
 fn create_random_ships() -> [Ship; 5] {
     // randomly place 5 ships on the board
     let mut rng = thread_rng();
-    let mut ships = [Ship::default(); 5];
+    let mut ships : [Ship; 5] = array_init::array_init(|_| Ship::default());
```

Comment:
@flaub I made an update to remove `Copy`. This was the line that was complaining. I saw a few ways to get around this, but opted for a safe wrapper called `array_init`. Let me know what you think of this approach.

Reply:
LGTM